tools/editor/create_dialog.cpp

```diff
@@ -107,6 +108,9 @@ void CreateDialog::add_type(const String& p_type,HashMap<String,TreeItem*>& p_ty
 
 	}
 
+        const String& description = EditorHelp::get_doc_data()->class_list[p_type].brief_description;
```

Comment:
Can you give me a quick hint how to do this in git? Normally I would commit a new version and then rebase-squash the two change sets. "amend the commit" sounds like there is an easier way, or am I reading too much into it?